src/Timers.h

```diff
@@ -9,6 +9,30 @@
 #include <ctime>
 #include <type_traits>
 
+#define AUTOTIMER_SEARCH_CASE_SENSITIVE "sensitive"
+#define AUTOTIMER_SEARCH_CASE_INSENITIVE ""
+
+#define AUTOTIMER_ENABLED_YES "yes"
+#define AUTOTIMER_ENABLED_NO "no"
+
+#define AUTOTIMER_ENCODING "UTF-8"
+
+#define AUTOTIMER_SEARCH_TYPE_EXACT "exact"
+#define AUTOTIMER_SEARCH_TYPE_DESCRIPTION "description"
+#define AUTOTIMER_SEARCH_TYPE_START "start"
+#define AUTOTIMER_SEARCH_TYPE_PARTIAL ""
+
+#define AUTOTIMER_AVOID_DUPLICATE_DISABLED ""                       //Require Description to be unique - No 
+#define AUTOTIMER_AVOID_DUPLICATE_SAME_SERVICE "1"                  //Require Description to be unique - On same service
+#define AUTOTIMER_AVOID_DUPLICATE_ANY_SERVICE "2"                   //Require Description to be unique - On any service
+#define AUTOTIMER_AVOID_DUPLICATE_ANY_SERVICE_OR_RECORDING "3"      //Require Description to be unique - On any service/recording
+
+#define AUTOTIMER_CHECK_SEARCH_FOR_DUP_IN_TITLE "0"                 //Check for uniqueness in - Title
+#define AUTOTIMER_CHECK_SEARCH_FOR_DUP_IN_TITLE_AND_SHORT_DESC "1"  //Check for uniqueness in - Title and Short description
+#define AUTOTIMER_CHECK_SEARCH_FOR_DUP_IN_TITLE_AND_ALL_DESCS ""    //Check for uniqueness in - Title and all descpritions
+
+#define AUTOTIMER_DEFAULT ""
+#define AUTOTIMER_DEFAULT_OMIT(a) (a == AUTOTIMER_DEFAULT)
```

Comment:
You should prefer static constants over `#defines`, because of type safety, C++ vs C, etc.